components/eam/src/physics/cam/ndrop.F90

```diff
@@ -1230,101 +1233,71 @@ subroutine activate_modal(wbar, sigw, wdiab, wminf, wmaxf, tair, rhoair,  &
 
    !      input
 
-   real(r8), intent(in) :: wbar          ! grid cell mean vertical velocity (m/s)
-   real(r8), intent(in) :: sigw          ! subgrid standard deviation of vertical vel (m/s)
-   real(r8), intent(in) :: wdiab         ! diabatic vertical velocity (0 if adiabatic)
-   real(r8), intent(in) :: wminf         ! minimum updraft velocity for integration (m/s)
-   real(r8), intent(in) :: wmaxf         ! maximum updraft velocity for integration (m/s)
-   real(r8), intent(in) :: tair          ! air temperature (K)
-   real(r8), intent(in) :: rhoair        ! air density (kg/m3)
-   real(r8), intent(in) :: na(:)      ! aerosol number concentration (/m3)
+   real(r8), intent(in) :: wbar          ! grid cell mean vertical velocity [m/s]
+   real(r8), intent(in) :: wmaxf         ! maximum updraft velocity for integration [m/s]
+   real(r8), intent(in) :: tair          ! air temperature [K]
+   real(r8), intent(in) :: rhoair        ! air density [kg/m3]
+   real(r8), intent(in) :: na(:)      ! aerosol number concentration [#/m3]
    integer,  intent(in) :: nmode      ! number of aerosol modes
-   real(r8), intent(in) :: volume(:)  ! aerosol volume concentration (m3/m3)
-   real(r8), intent(in) :: hygro(:)   ! hygroscopicity of aerosol mode
+   real(r8), intent(in) :: volume(:)  ! aerosol volume concentration [m3/m3]
+   real(r8), intent(in) :: hygro(:)   ! hygroscopicity of aerosol mode [dimensionless]
```

Comment:
should it be "unitless" other than "dimensionless"?

Reply:
OK, will just take out everything that says [dimensionless] (though will retain m3/m3 type of units for clarity).

Reply:
Sorry, I misinterpreted the question.  Will change the word 'dimensionless' to 'unitless', thanks.